src/util/update.ts

```diff
@@ -0,0 +1,6 @@
+import {User} from "../entity/User"
```

Comment:
I'd give this file a better name, like `updateName`. You could also put it in the Discord utils file.